neuralmonkey/decoders/transformer.py

```diff
@@ -124,6 +126,18 @@ def output_dimension(self) -> int:
 
     def embed_inputs(self, inputs: tf.Tensor) -> tf.Tensor:
         embedded = tf.nn.embedding_lookup(self.embedding_matrix, inputs)
+
+        if (self.embeddings_source is not None
+                and (self.embeddings_source.multiply_embedding_mode
+                     == "sqrt_depth")):
```

Comment:
co s tadytim humusem? Přece tam nemůžeme mít místo boolean vlaječky stringovou proměnnou, která je buď `None` nebo `"sqrt_depth"` a nic jinýho neni validní..

Reply:
Nazval bych to `scale_by_size_squared: bool = False`

Reply:
A pak az nas napadne dalsi zpusob scalovani, tak to budem znovu prepisovat. Ja bych to nechal tak.

Reply:
To je blbost, za prvý nás další způsob nenapadne, což je hlavní argument, a za druhý, i kdyby nás napad, tak se přidá `scale_by_whatever_comes_next: bool = taky False` a bude se kontrolovat že jen jedno je `True`.